backend/src/akvo/lumen/lib/aggregation/bar.clj

```diff
@@ -1,67 +1,34 @@
 (ns akvo.lumen.lib.aggregation.bar
   (:require [akvo.lumen.lib :as lib]
-            [akvo.lumen.postgres.filter :as filter]
-            [akvo.lumen.lib.dataset.utils :as utils]
+            [akvo.lumen.lib.dataset.utils :refer (find-column)]
+            [clojure.pprint :refer (pprint)]
+            [clojure.tools.logging :as log]
+            [akvo.lumen.postgres.filter :refer (sql-str)]
             [clojure.java.jdbc :as jdbc]))
 
-(defn- run-query [tenant-conn table-name sql-text column-x-name column-y-name filter-sql aggregation-method truncate-size column-subbucket-name]
-  (rest (jdbc/query tenant-conn
-                    [(format sql-text
-                             column-x-name column-y-name table-name filter-sql aggregation-method truncate-size column-subbucket-name)]
-                    {:as-arrays? true})))
+(defn- run-query [tenant-conn q]
```

Comment:
All ns have one of this fn. Is it common?

Reply:
it was common ...